packages/webpack-config-anansi/src/base/index.js

```diff
@@ -184,6 +186,7 @@ export default function makeBaseConfig({
         'node_modules',
       ],
       extensions: ['.js', '.ts', '.tsx', '.scss', '.json'],
+      alias: webpack.version.startsWith('4') ? {} : NODE_ALIAS,
```

Comment:
seems more reliable to use the `semver` library here, but i suppose by the time webpack 40 comes out, this code will be different :-p

Reply:
exactly. deprecating 4 support will happen long before then :P